.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # Every request must be reviewed and accepted by:
 
-*	@felangel
\ No newline at end of file
+*	@felangel @jeroen-meijer
```

Comment:
Should we add a newline here

Reply:
I wish formatting did so by default... Let's do it.